specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1346 @@
+            "description": "Collection of documents to analyze.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MultiLanguageBatchInput"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call results in a document sentiment prediction, as well as sentiment scores for each sentiment class (Positive, Negative, and Neutral)",
+            "schema": {
+              "$ref": "#/definitions/SentimentResponse"
+            }
+          },
+          "default": {
+            "description": "Error Response",
+            "schema": {
+              "$ref": "#/definitions/TextAnalyticsError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Sentiment request": {
+            "$ref": ".//examples//SuccessfulSentimentRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "MultiLanguageBatchInput": {
+      "type": "object",
+      "required": [
+        "documents"
+      ],
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "The set of documents to process as part of this batch.",
+          "items": {
+            "$ref": "#/definitions/MultiLanguageInput"
+          }
+        }
+      },
+      "description": "Contains a set of input documents to be analyzed by the service."
+    },
+    "MultiLanguageInput": {
+      "type": "object",
+      "required": [
+        "id",
+        "text"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A unique, non-empty document identifier."
+        },
+        "text": {
+          "type": "string",
+          "description": "The input text to process."
+        },
+        "language": {
+          "type": "string",
+          "description": "(Optional) This is the 2 letter ISO 639-1 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default."
+        }
+      },
+      "description": "Contains an input document to be analyzed by the service."
+    },
+    "DocumentError": {
+      "type": "object",
+      "required": [
+        "id",
+        "error"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Document Id."
+        },
+        "error": {
+          "type": "object",
+          "description": "Document Error.",
+          "$ref": "#/definitions/TextAnalyticsError"
+        }
+      }
+    },
+    "TextAnalyticsError": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "invalidRequest",
+            "invalidArgument",
+            "internalServerError",
+            "serviceUnavailable"
```

Comment:
Are there any new additions we want to add here?

Reply:
I dont think we have added anything. I will double check

Reply:
No we dont have anything